src/LinuxEvent.Tests/EventParseTests.cs

```diff
@@ -95,6 +93,16 @@ public void OneStack()
 			});
 		}
 
+        [Fact]
+        [UseCulture("fr-FR")]
```

Comment:
❓ I'm assuming you ran this test on your machine without your code change, and the test failed due to the described localization problem. Can you confirm that? I will try it locally as well. :smile: